qemu/tests/cfg/blockdev_inc_backup_expose_active_bitmap.cfg

```diff
@@ -50,7 +50,7 @@
     gluster_direct:
         enable_gluster_data1 = yes
     ceph:
-        enable_ceph_data1 = yes
```

Comment:
Let's keep this line for this case, I saw we explicitly enabled network storage for data1, which, I guessed, might be used for backing/rebase or something else, you know that sometimes we have to define image specified params explicitly to avoid errors.

Reply:
Have updated, thanks zhenchao.